### PR TITLE
Create Line Chart for Official Grades

### PR DESCRIPTION
**Pull request by AkshayPall**
Screen Shot 2021-06-29 at 8 52 06 AM

### CODE REVIEW

**Review by aravind-segu — APPROVED** [review]
LGTM

File: main.ts [hunk 1]
```diff
@@ -493,6 +493,12 @@ function renderAcademics(options) {
       return ((d) * 10) + '%'
     }
   });
+  renderLineChart(d3.select('#grades-official'), GRADES_OFFICIAL, options.width, 300, {
+    toggle: 'gender-salary',
```
Comment (jaxonL) [inline main.ts]: curious about this toggle -- what does it do?

**Review by jaxonL — COMMENTED** [review]
looks good; just a clarification question before approving

File: main.ts (hunk 1 above)
Comment (roxelo) [reply main.ts]: @jaxonL Oh I think that's from [my change here](https://github.com/sexxis/classprofile/pull/50) actually. I made some change to the line graph so you can have them interactive where needed. The toggle specifies what the class name for each line is for the purpose of toggling between visible and hidden. (for line 1, the class would be `gender-salary-0`)
I think with my change you need to provide a toggle field now, but maybe you don't I can't really remember what I did :/

**Review by roxelo — APPROVED** [review]

**Pull request merged**